.gitignore

```diff
@@ -0,0 +1,3 @@
+#node_modules
+.github/workflow/linters.yml
```

Comment:
- [ ] Please 🙏🏽 quit this folder from the .gitignore file. The linter should stay in the root folder.